packages/@ember/-internals/glimmer/lib/component.ts

```diff
@@ -26,6 +28,11 @@ import {
   IS_DISPATCHING_ATTRS,
 } from './component-managers/curly';
 
+// Keep track of which component classes have already been processed for lazy event setup.
+// Using a WeakSet would be more appropriate here, but this can only be used when IE11 support is dropped.
+// Thus the workaround using a WeakMap<object, true>
+let lazyEventsProcessed = new WeakMap<EventDispatcher, WeakMap<object, true>>();
```

Comment:
FWIW, we have a WeakSet polyfill thingy that we use (which lets us treat it like a `WeakSet`, but on IE11 is implemented as a `WeakMap`).

```js
import { _WeakSet as WeakSet } from '@glimmer/util';
```

https://github.com/glimmerjs/glimmer-vm/blob/master/packages/@glimmer/util/lib/weak-set.ts